Makefile

```
@@ -40,7 +40,13 @@ IMAGE_REGISTRY ?= local
 STAGING_REGISTRY := gcr.io/k8s-staging-provider-azure
 K8S_VERSION ?= v1.15.0
 HYPERKUBE_IMAGE ?= gcrio.azureedge.net/google_containers/hyperkube-amd64:$(K8S_VERSION)
-IMAGE_TAG ?= $(shell git rev-parse --short=7 HEAD)
+
+ifndef TAG
+	IMAGE_TAG = $(shell git rev-parse --short=7 HEAD) 
```

```suggestion
	IMAGE_TAG ?= $(shell git rev-parse --short=7 HEAD) 
```
Using a conditional variable assignment operator is necessary to pick up environment variables set in https://github.com/kubernetes/test-infra/blob/master/kubetest/azure.go#L885-L887, which are used to build customm ccm and cnm according to the PR and run e2e test with them.

---

Thank you @chewong .